defs.bzl

```diff
@@ -24,7 +24,8 @@ load("//internal/karma:ts_web_test.bzl",
      _ts_web_test = "ts_web_test_macro",
      _ts_web_test_suite = "ts_web_test_suite")
 load("//internal/protobufjs:ts_proto_library.bzl", _ts_proto_library = "ts_proto_library")
-load("//:package.bzl", _check_rules_typescript_version = "check_rules_typescript_version")
+load("//:package.bzl", "VERSION")
```

Comment:
yeah, the difference is that rules_nodejs doesn't need an external repo to load the check function, so it can be together. in rules_typescript we can't have a load statement in the package.bzl file since it's responsible for fetching the rules_nodejs repo.

We could change rules_nodejs to have the check function outside the package.bzl.